Classes/Response/YLPLocation.m

```diff
@@ -11,24 +11,20 @@
 
 @implementation YLPLocation
 
-- (instancetype)initWithDictionary:(NSDictionary *)location {
+- (instancetype)initWithDictionary:(NSDictionary *)location coordinate:(YLPCoordinate *)coordinate {
     if (self = [super init]) {
         _city = location[@"city"];
-        _stateCode = location[@"state_code"];
-        _postalCode = location[@"postal_code"];
-        _countryCode = location[@"country_code"];
-        _crossStreets = location[@"cross_streets"];
+        _stateCode = location[@"state"];
+        _postalCode = location[@"zip_code"];
```

Comment:
Same here, trying to minimize breaking changes so that we can get as many people off APIv2 as possible.